scalafix/rules/src/main/scala/rsc/rules/pretty/SemanticdbPrinter.scala

```diff
@@ -142,9 +146,15 @@ class SemanticdbPrinter(
               rep(" ", anns, " ", "")(pprint)
           }
         case s.ExistentialType(utpe, decls) =>
-          decls.infos.foreach(symbols.append)
-          opt(utpe)(normal)
-          rep(" forSome { ", decls.infos, "; ", " }")(pprint)
+          if (config.better) {
+            val wildcardInfos = decls.infos.map(_.withDisplayName("_"))
```

Comment:
There are cases where it’s not safe to convert existentials into wildcards, for example “Map[T, T] forSome { type T }”. I think the rule is that existential types can only be converted into wildcards if they’re referenced once.